app/Http/Controllers/User/CalendarController.php

```diff
@@ -15,46 +15,46 @@ class CalendarController extends Controller
 {
     public function index()
     {
-    	$hours = range(0, 23);
-    	$today = Carbon::now()->format('Y-m-d');
+        $hours = range(0, 23);
+        $today = Carbon::now()->format('Y-m-d');
 
-    	return view('user.teacher.calendar', compact('hours', 'today'));
+        return view('user.teacher.calendar', compact('hours', 'today'));
     }
 
     public function update(Request $request)
     {
-    	$user = Auth::user();
+        $user = Auth::user();
 
-    	if ($user->role != 'teacher' || !$user->teacher) {
-    		return abort(Response::HTTP_FORBIDDEN);
-    	}
+        if ($user->role != 'teacher' || !$user->teacher) {
+            return abort(Response::HTTP_FORBIDDEN);
+        }
 
-    	$inputs = collect($request->all())->groupBy('date');
+        $inputs = collect($request->all())->groupBy('date');
 
-    	foreach ($inputs as $date => $value) {
-    		$schedule = Schedule::create([
-    			'date' => $date,
-    			'schedule_type' => Teacher::class,
-    			'schedule_id'=> $user->teacher->id,
-    		]);
+        foreach ($inputs as $date => $value) {
+            $schedule = Schedule::create([
+                'date' => $date,
+                'schedule_type' => Teacher::class,
+                'schedule_id'=> $user->teacher->id,
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/CalendarController.php</br>**Line**: 38</br>Expected 1 space between "'schedule_id'" and double arrow; 0 found